backend/src/main/java/com/eyetrade/backend/service/ArticleService.java

```diff
@@ -120,6 +120,17 @@ public ArticleResource updateArticle(UUID userId, ArticleDto articleDto, UUID ar
         }
     }
 
+    public ArticleResource deleteArticle(UUID userId, UUID articleId) throws IllegalAccessException {
```

Comment:
It should be transactional because if it is interrupted at the middle of process there will be ruined datas